app/src/main/java/com/orange/ods/app/ui/components/listitem/ComponentListItem.kt

```diff
@@ -0,0 +1,236 @@
+    }
+
+    Subtitle(textRes = R.string.component_list_trailing, horizontalPadding = true)
+    OdsChoiceChipsFlowRow(
+        selectedChip = listItemCustomizationState.selectedTrailing,
+        modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin)),
+        outlinedChips = true
+    ) {
+        listItemCustomizationState.trailings.forEach { trailing ->
+            OdsChoiceChip(textRes = trailing.textResId, value = trailing)
+        }
+    }
+
+}
+
+@Composable
+private fun ComponentListsContent(listItemCustomizationState: ListItemCustomizationState) {
+    val recipes = LocalRecipes.current.filter { it.description.isNotBlank() }
+    with(listItemCustomizationState) {
+        Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
+            if (!trailings.contains(selectedTrailing.value)) {
+                resetTrailing()
+            }
+
+            val modifier = Modifier
+                .let { modifier ->
+                    iconType?.let { modifier.iconType(it) }.orElse { modifier }
+                }
+            val singleLineSecondaryText = lineCount.value == 2
+
+            val text = recipes.first().title
+            val secondaryText = listItemCustomizationState.getSecondaryText(recipes.first())
+            val icon: @Composable (OdsListItemIconScope.() -> Unit)? =
+                listItemCustomizationState.getIconPainter(recipes.first())?.let { { OdsListItemIcon(painter = it) } }
+            trailing?.let { listItemTrailing ->
+                OdsListItem(
+                    modifier = modifier,
+                    text = text,
+                    secondaryText = secondaryText,
+                    singleLineSecondaryText = singleLineSecondaryText,
+                    icon = icon,
+                    trailing = listItemTrailing
+                )
+            }.orElse {
+                val context = LocalContext.current
+                OdsListItem(
+                    modifier = modifier.clickable { clickOnElement(context = context, context.getString(R.string.component_list_item)) },
+                    text = text,
+                    secondaryText = secondaryText,
+                    singleLineSecondaryText = singleLineSecondaryText,
+                    icon = icon
+                )
+            }
+            CodeImplementationColumn(modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin))) {
+                FunctionCallCode(
+                    name = OdsComposable.OdsListItem.name,
+                    parameters = {
+                        string("text", text)
+                        if (singleLineSecondaryText) secondaryText?.let { string("secondaryText", it) }
+                        icon()
+                    }
+                )
+            }
+        }
+    }
+}
+
+private val ListItemCustomizationState.Trailing.textResId: Int
```

Comment:
Starting from this line to the end of the line, all code is duplicated from `ComponentLists`. We should factorize all this code.

But maybe we don't even need this in `ComponentLists` in fact. Do we have to keep `List item size`, `Leading` and `Trailing` in the customization bottom sheet of `ComponentLists`? It's already in `ComponentListItem`so I would say it is not needed.